content/articles/entra-identity-provider.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Entra as an Identity Provider
+excerpt: How to use Entra to create and log in to accounts.
+meta: Learn how to seamlessly integrate Entra as your identity provider for account creation and secure logins, enhancing your user experience and security.
+categories:
+- Account
+---
+
+# Entra as an Identity Provider
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+Using Entra as an identity provider for single sign-on streamlines the login experience for you and your team and helps you manage risk at scale.
+
+<info>
+Entra as an Identity Provider is only available on Enterprise plans. See our [pricing page](https://dnsimple.com/pricing) for more details.
+</info>
+
+## Prerequisites
+
+To proceed with configuring login with SSO through Entra, you must:
+
+- Have [Application Administrator](https://learn.microsoft.com/en-us/entra/identity/role-based-access-control/permissions-reference#application-administrator) access to a Microsoft Entra account.
+- Have administrator access to a DNSimple account.
+- Be subscribed to a DNSimple [Enterprise plan](https://dnsimple.com/enterprises).
+
+## Linking an Entra organization to a DNSimple account
+
+DNSimple's Entra app integration is pending review and is not yet available for installation via the [Microsoft Entra App Gallery](https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-integrated-apps).
+
+### Creating a customer Entra app
+
+For now, you can use Entra as an Identity Provider by [creating your own custom **Single tenant** Entra app integration](https://docs.azure.cn/en-us/data-explorer/kusto/access-control/provision-entra-id-app?view=microsoft-fabric&tabs=portal) to work with DNSimple by following these steps. Take note of your **Directory (tenant) ID**, **Application (client) ID**, and **Client secret**.
+
+### Adding Redirect URIs
+
+After creating your app, add the [Redirect URIs](https://learn.microsoft.com/en-us/entra/identity-platform/how-to-add-redirect-uri) for your new web app using the following URIs:
+1. https://dnsimple.com/identity_providers/entra/callbacks/users/login
+1. https://dnsimple.com/identity_providers/entra/callbacks/accounts/link
+1. https://dnsimple.com/identity_providers/entra/callbacks/users/link
+
+### Adding API permissions
+
+Add the following API Permissions to the app and grant access:
+
+- openid
+- email
+- profile
+- User.Read
+- User.Read.All
```

Comment:
```suggestion
Add the following [API Permissions](https://learn.microsoft.com/en-us/entra/identity-platform/howto-update-permissions) to the app:

- openid
- email
- profile
- User.Read
- User.Read.All
```

You'll also need to [grant consent](https://learn.microsoft.com/en-us/entra/identity-platform/howto-update-permissions?pivots=portal#grant-consent-for-the-added-permissions-for-the-enterprise-application) for these permissions.